### PR TITLE
Language overhaul 3

### PR DESCRIPTION
**Pull request by 3gf8jv4dv**
Continue with the overhaul of en-US. Hopefully this will be my last overhaul, with only minor revisions to follow.

Take some opinions from Chiloven945 ([Chiloven945@`509469b`](https://github.com/Chiloven945/PCL2/commit/509469be3b0c7e9df1646c0ab1c47d21647715d0)). Thanks!

### CODE REVIEW

**Comment by Chiloven945** [comment]
Thanks for taking my opinions! I will check it stat. and do more modification.

**Review by tangge233 — APPROVED** [review]
Nice work.

**Comment by Chiloven945** [comment]
I've done some new changes in my fork, sorry but I don't have enough time. I'll do more later.

File: Plain Craft Launcher 2/Pages/PageDownload/ModDownloadLib.vb [hunk 1]
```diff
@@ -1645,12 +1645,12 @@ Retry:
         '显示各个版本
         If FreshStableVersion IsNot Nothing Then
             Dim Fresh = NeoForgeDownloadListItem(FreshStableVersion, OnClick, IsSaveOnly)
-            Fresh.Info = If(Fresh.Info = "", GetLang("LangDownloadNewStable"), GetLang("LangDownloadLatest") & Fresh.Info)
+            Fresh.Info = If(Fresh.Info = "", GetLang("LangDownloadNewStable"), GetLang("LangDownloadNewStable"))
```
Comment (tangge233) [inline Plain Craft Launcher 2/Pages/PageDownload/ModDownloadLib.vb:1648]: Why delete "Fresh.Info" here?

File: Plain Craft Launcher 2/Pages/PageDownload/ModDownloadLib.vb [hunk 2]
```diff
@@ -1645,12 +1645,12 @@ Retry:
...
             Stack.Children.Add(Fresh)
         End If
         If FreshBetaVersion IsNot Nothing Then
             Dim Fresh = NeoForgeDownloadListItem(FreshBetaVersion, OnClick, IsSaveOnly)
-            Fresh.Info = If(Fresh.Info = "", GetLang("LangDownloadNewTest"), GetLang("LangDownloadLatest") & Fresh.Info)
+            Fresh.Info = If(Fresh.Info = "", GetLang("LangDownloadNewTest"), GetLang("LangDownloadNewTest"))
```
Comment (tangge233) [inline Plain Craft Launcher 2/Pages/PageDownload/ModDownloadLib.vb:1653]: Also here

**Review by tangge233 — CHANGES_REQUESTED** [review]
A small problem.

**Comment by tangge233** [comment]
I'm sorry, that was my mistake.Nothing wrong actually.

**Pull request merged**